metric/client/client_test.go

```
@@ -209,6 +211,62 @@ var _ = Describe("Client", func() {
 				})
 			})
 
+			Context("as service with service secret", func() {
+				var serverSessionToken string
+
+				BeforeEach(func() {
+					serverSessionToken = test.RandomStringFromRangeAndCharset(64, 64, test.CharsetAlphaNumeric)
+					ctx = log.NewContextWithLogger(ctx, logNull.NewLogger())
```

golangci-lint (2.13.2)

[error] nested context in function literal

(fatcontext)